### ISSUE TITLE
is there a bottom line?

### ISSUE THREAD

**Issue by OhadR** [anton-johansson/geospatial-mongodb-vs-elasticsearch#1]
i also search what is the best fit for geospatial - ES of mongo. what is the bottom line? which one is better?

thanks!

**Comment by anton-johansson** [comment]
@OhadR, nah, I never got around to finish the test. The product I was planning on creating was dropped too. :)
